types/yauzl-promise/package.json

```diff
@@ -1,13 +1,12 @@
 {
     "private": true,
     "name": "@types/yauzl-promise",
-    "version": "2.1.9999",
+    "version": "3.0.9999",
```

Comment:
the title says '4', but here the version is v3.

Reply:
The newest version of yauzl-promise is v4, but I thought I should just bump the @types dependency by a major version.
Should I try to match yauzl-promises version?

Reply:
Yes, please, there is no need for version consistency here. With DT, latest available is matched

Reply:
Is now as close as I can get to upstream version